frontend/src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
?

Reply:
jeśli dobrze widzę to to jest https://github.com/facebook/create-react-app/issues/6560

Reply:
Tak ale to bezuzyteczny plik na ten moment wiec do wywalenia imo